### DISCUSSION TITLE
Word Comment Replay get as the DOCX convert

### DISCUSSION THREAD

**Discussion by PerlDeveloperSurya** [jgm/pandoc#10155] [General]
how to render the comment replay as the HTML to DOCX

**Comment by PerlDeveloperSurya** [comment]
![Uploading image.png…]()

**Comment by jgm** [comment]
I do not understand the question.

**Comment by PerlDeveloperSurya** [comment]
reply added the already added comment

**Comment by jgm** [comment]
Sorry, I still do not understand what you are asking about.

**Comment by PerlDeveloperSurya** [comment]
[commet.docx](https://github.com/user-attachments/files/16958626/commet.docx)
Refer to the attached document.

**Comment by PerlDeveloperSurya** [comment]
![image](https://github.com/user-attachments/assets/7b73e354-2498-48c2-80e9-9dd0b2ccf58c)

**Reply by jgm** [reply]
Sorry, I still don't understand. You need to actually explain in words what you are doing (exact command line, input), what result you are seeing, and how it differs from what you expect.

**Comment by PerlDeveloperSurya** [comment]
I already inserted the comment (Insert->Comments->Comment) in the word file. In this comment, I have added a replay of the already-added comment. Refer to the above screenshot and word file.
